src/device.rs

```diff
@@ -63,9 +63,10 @@ pub(crate) struct Config<'a> {
     pub product_id: u16,
     pub usb_rev: UsbRev,
     pub device_release: u16,
-    pub manufacturer: Option<&'a str>,
-    pub product: Option<&'a str>,
-    pub serial_number: Option<&'a str>,
+    pub extra_lang_ids: Option<&'a [u16]>,
+    pub manufacturer: Option<&'a [&'a str]>,
+    pub product: Option<&'a [&'a str]>,
+    pub serial_number: Option<&'a [&'a str]>,
```

Comment:
I'm assuming the order of entries in the lists corresponds to the order in `extra_lang_ids`. If so, we need to specify that somehow to the user.

Reply:
I have doc the relationship of 4 lists in 3 methods of the `UsbDeviceBuilder`. Should I also put a comment on `Config` struct too?

Reply:
I saw the documentation after reviewing this, I think what we have already is appropriate